src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -737,11 +737,11 @@ def _set_dserial_params(self, src=None, dst_pool=None):
         # Get an intermediate path for HDF5 file(s)
         tmp_path = self.new_posix_test_path(create=False, parent=self.serial_tmp_dir)
 
-        # Set the source params for dserialize
+        # Set the source params for serialize
```

Comment:
I think this was probably correct, at least according to the methods.  Frankly I'm far from sure though.